src/Symfony/Component/VarDumper/Tests/Caster/IntlCasterTest.php

```diff
@@ -262,7 +262,8 @@ public function testCastDateFormatter()
         $expectedTimeZoneRawOffset = $expectedTimeZone->getRawOffset();
         $expectedTimeZoneDSTSavings = $expectedTimeZone->getDSTSavings();
 
-        $expected = <<<EOTXT
+        if ($expectedTimeZone->useDaylightTime() === true) {
```

Comment:
` === true` is useless. The method already returns a boolean. Using a strict comparison to turn it again into a boolean (the same one) does not bring any value.